### ISSUE TITLE
"The method or operation is not implemented." when trying the seed operation...

### ISSUE THREAD

**Issue by rutgoff** [matthew-humphrey/nmssavetool#10]
Hello,
Excellent program; have been enjoying it's use a lot recently. Came across an issue with the seed option, see below.

System info 
OS: Windows 10 Pro
Version: 10.0.15063 Build 15063
Processor:  Intel(R) Core(TM) i7-7700K CPU @ 4.20GHz, 4200 Mhz, 4 Core(s), 8 Logical Processor(s)
Installed Physical Memory (RAM): 64.0 GB

Game Info
Source: Steam
Version: 1.37

```
PS C:\Users\[RDACTED]\Desktop\nmssavetool-2.0.0> .\nmssavetool.exe seed -g normal -c ship -r
The method or operation is not implemented.
PS C:\Users\[RDACTED]\Desktop\nmssavetool-2.0.0> .\nmssavetool.exe seed -g normal -c ship -r
The method or operation is not implemented.
PS C:\Users\[RDACTED]\Desktop\nmssavetool-2.0.0> .\nmssavetool.exe seed -g normal -c ship -s 0xde03fa3b29c16365
The method or operation is not implemented.
PS C:\Users\[RDACTED]\Desktop\nmssavetool-2.0.0>
```

Happy to provide any other necessary info to help you diagnose this.

**Comment by matthew-humphrey** [comment]
Thanks! This is fixed in version 2.0.1.1-alpha. Unfortunately, that version is not backwards compatible with NMS 1.37. It has some updates to support the new save game format coming in 1.38. IF you're on the NMS experimental branch, you can go ahead and download the pre-release below, otherwise it will be released shortly after NMS 1.38 goes live.

https://github.com/matthew-humphrey/nmssavetool/releases/tag/v2.0.1.1-alpha

**Comment by matthew-humphrey** [comment]
BTW, I wouldn't mind fixing this in the old version, but I don't have a way to test it currently because my NMS is already running the experimental branch.
